src/components/About/Contact/Contact.react.js

```diff
@@ -63,23 +95,37 @@ const Contact = props => {
               through various channels."
       />
       <Section>
-        <ContactContent>
-          <H5>SUSI</H5>
-          <Text>
-            12 Eu Tong Sen Street
-            <br />
-            #8-169 ,
-            <br />
-            Singapore 059819
-            <br />
-            Phone +65 83435672
-            <br />
-            Email: support@susper.net
-            <br />
-            Board of Directors: Phuc Hong Dang
-            <br />
-          </Text>
-        </ContactContent>
+        <Address>
+          <div>
+            <H5>SUSI</H5>
+            <Text>
+              12 Eu Tong Sen Street
+              <br />
+              #8-169 ,
+              <br />
+              Singapore 059819
+              <br />
+              Phone +65 83435672
+              <br />
+              Email: support@susper.net
+              <br />
+              Board of Directors: Phuc Hong Dang
+              <br />
+            </Text>
+          </div>
+          <MapBox>
+            {mapKey && (
+              <MapContainer
+                google={google}
+                style={{
+                  width: '65vw',
+                }}
+                tooltipRender={selectedPlace => toolTip(selectedPlace)}
```

Comment:
Keep the name of this prop as `tooltipRenderer`
- `tooltipRenderer={renderTooltip}` would also work here.